README.md

```diff
@@ -9,7 +9,7 @@ Follow the [Getting Started](https://docs.microsoft.com/aspnet/core/getting-star
 
 Also check out the [.NET Homepage](https://www.microsoft.com/net) for released versions of .NET, getting started guides, and learning resources.
 
-See the [Issue Management Policies](https://github.com/dotnet/aspnetcore/blob/master/docs/IssueManagementPolicies.md) document for more information on how we handle incoming issues.
+See the [Triage Process](https://github.com/dotnet/aspnetcore/blob/master/docs/TriageProcess.md) document for more information on how we handle incoming issues.
```

Comment:
Why do we have these two separate docs: IssueManagementPolicies.md vs TriageProcess.md? Should they be consolidated into one?

Reply:
One is about the automations we have. I thought about this and this doesn't fit into the `triage process`. Removing this is an option, but I didn't want to remove yet. Maybe in the future.